cms/check_used_space

```diff
@@ -0,0 +1,99 @@
+                attributes = list_rse_attributes(rse['id'])
+                country = attributes.get('country', 'UNKNOWN')
+                rse_type = session.query(models.RSE.rse_type).filter(models.RSE.id == rse['id']).scalar()
+                rse_type = str(rse_type).split('.', 1)[1]
+                limits = get_rse_limits(rse['id'])
+                rucio_used = None
+                static_used = None
+                free_space = None
+                prom_labels = {'rse': rse['rse'], 'country': country, 'rse_type': rse_type, 'source': ''}
+                for usage in sources:
+                    
+                    # Calculate free rucio space of RSE and push it
+                    if usage['source'] == 'rucio':
+                        prom_labels['source'] = 'rucio_free_space'
+                        rucio_used = usage['used']
+                        rucio_free_space = int(usage['total']) - int(usage['used'])
+                        (manager.gauge(name='report_used_space.{rse}.{country}.{rse_type}.rucio_free_space',
+                                   documentation='Space used at an RSE from various sources')
+                         .labels(**prom_labels)
+                         .set(rucio_free_space))
+                        print(rse['rse'], country, rse_type, 'rucio_free_space', rucio_free_space)    
+                    
+                    # Calculate total free space of RSE (static-rucio) and push it
+                    if usage['source'] == 'static':
+                        static_used = usage['used']
+                    
+                    if rucio_used and static_used:
+                        prom_labels['source'] = 'free_space'
+                        free_space = int(static_used) - int(rucio_used)
+                        (manager.gauge(name='report_used_space.{rse}.{country}.{rse_type}.free_space',
+                                   documentation='Space used at an RSE from various sources')
+                         .labels(**prom_labels)
+                         .set(free_space))
+                        print(rse['rse'], country, rse_type, 'free_space', free_space)   
+                        
+                    source = usage['source']
+                    prom_labels['source'] = source    
+                    (manager.gauge(name='report_used_space.{rse}.{country}.{rse_type}.{source}',
+                                   documentation='Space used at an RSE from various sources')
+                     .labels(**prom_labels)
+                     .set(usage['used']))
+                    print(rse['rse'], country, rse_type, source, usage['used'])
+                      
+                # export and push `minfreespace` value from RSE limits
+                if limits.get('MinFreeSpace'):
+                    prom_labels['source'] = 'minfreespace'
+                    (manager.gauge(name='report_used_space.{rse}.{country}.{rse_type}.minfreespace',
```

Comment:
This should be min_free_space to be consistent. Then I think the full name will be "rucio_probes_min_free_space" which seems descriptive enough.

Reply:
Thanks Eric, I've changed all the names. Just to make sure, the `rucio_probes_` will come from the base gauge implementation , right?